app/res/values/themes.xml

```diff
@@ -74,8 +74,8 @@
     <style name="ActionBar.TitleTextStyle">
         <item name="android:textColor">@color/white</item>
         <item name="android:paddingLeft">16dp</item>
-        <item name="android:textSize">18sp</item>
-        <item name="android:textStyle">bold</item>
+        <item name="android:textSize">22sp</item>
+        <item name="android:fontWeight" tools:targetApi="o">400</item>

```

Comment:
Do we not need this on pre O devices ?

Reply:
we need it on all but weight 400 is available above this only otherwise we have only bold option

Reply:
Yes we need it but font weight 400 doses not support below that , we have to use bold or have to get ttf file

Reply:
can we use bold instead then to be consistent across devices ?